pkg/api/vlabs/types.go

```diff
@@ -125,6 +125,7 @@ type OrchestratorProfile struct {
 // Kubernetes specific configuration
 type KubernetesConfig struct {
 	KubernetesImageBase string `json:"kubernetesImageBase,omitempty"`
+	NetworkPolicy       string `json:"networkPolicy,omitempty"`
```

Comment:
Based on discussion in this PR looks like Calico is being used only for in host networking for now. But I think it could be expanded later to do host to host networking etc. Also, there might be a need for users to choose another plugin. 

With that in mind its best if the API is extensible for future network policy configurations. I recommend the  following approach:

```
type KubernetesConfig struct {
	KubernetesImageBase string 'json:"kubernetesImageBase,omitempty"'
	NetworkConfig NetworkConfig 'json:"networkConfig,omitempty"'
}

type NetworkConfig struct {
	NetworkPolicy       string 'json:"networkPolicy,omitempty"'
}
```